src/main/java/subway/section/Sections.java

```diff
@@ -0,0 +1,60 @@
+package subway.section;
+
+import subway.station.Station;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Embeddable
+public class Sections {
+    @OneToMany(mappedBy = "line", cascade = {CascadeType.PERSIST}, orphanRemoval = true)
+    private List<Section> sections = new ArrayList<>();
+
+    public void addSection(Section section) {
+        this.sections.add(section);
+    }
+
+    public void remove() {
+        sections.remove(getLastStationIndex());
+    }
+
+    public List<Station> getStations() {
+        List<Station> stations = sections.stream()
+                .map(Section::getDownStation)
+                .collect(Collectors.toList());
+        stations.add(0, sections.get(0).getUpStation());
+        return stations;
+    }
+
+    public boolean isEmpty() {
+        return sections.isEmpty();
+    }
+
+    public boolean isLastStation(Station station) {
+        return getLastStation().equals(station);
+    }
+
+    public Station getLastStation() {
+        return sections.get(getLastStationIndex()).getDownStation();
+    }
+
+    public boolean containsStation(Station station) {
+        return getStations().contains(station);
+    }
+
+    public boolean isSingleSection() {
+        return sections.size() == 1;
+    }
+
+    private int getLastStationIndex() {
+        return sections.size() - 1;
+    }
+
+    public int calcDistance() {
+        return sections.stream().mapToInt(Section::getDistance).sum();
```

Comment:
한줄에 다 나열하는 것보다 한 줄에 한 기능씩 나열하는게 코드 파악에 용이하지 않을까요?

```suggestion
        return sections.stream()
            .mapToInt(Section::getDistance)
            .sum();
```